### PR TITLE
provider/aws: Add proxy protocol support in ELB

### LINKED ISSUE #948: Support ELB proxy protocol
It would be useful to have control over the proxy protocol for ELB. See:
http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/enable-proxy-protocol.html#enable-proxy-protocol-api

### PR DESCRIPTION
**Pull request by nabeken**
This PR tries to fix #948 and I was inspired by #1685.

If you are okay with this, I'll work on the documentation, thanks.

### CODE REVIEW

**Comment by mitchellh** [comment]
This is perfect, can you add docs? Thanks!

**Comment by nabeken** [comment]
@mitchellh Thank you for your review. docs pushed.

File: website/source/docs/providers/aws/r/proxy_protocol_policy.html.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+layout: "aws"
+page_title: "AWS: aws_proxy_protocol_policy"
+sidebar_current: "docs-aws-proxy-protocol-policy"
+description: |-
+  Provides an proxy protocol policy, which allows an ELB to carry a client connection information to a backend.
+---
+
+# aws\_proxy\_protocol\_policy
+
+Provides an proxy protocol policy, which allows an ELB to carry a client connection information to a backend.
```
Comment (catsby) [inline website/source/docs/providers/aws/r/proxy_protocol_policy.html.markdown]: > Provides an proxy protocol policy

"Provides a proxy protocol policy"

Comment (nabeken) [inline website/source/docs/providers/aws/r/proxy_protocol_policy.html.markdown]: oops, my bad copy & paste:scream_cat: pushed.

**Comment by catsby** [comment]
Thanks!

**Pull request merged**